os/gcron/gcron_schedule.go

```diff
@@ -189,7 +188,9 @@ func parsePatternItem(item string, min int, max int, allowQuestionMark bool) (ma
 					return nil, gerror.NewCodef(gcode.CodeInvalidParameter, `invalid pattern item: "%s"`, item)
 				} else {
 					rangeMin = number
-					rangeMax = number
```

Comment:
You'd better commit an unit testing case for this update.

Reply:
The TestFunction of  "TestSlash"  in file gcron_schedule_test.go  is the unit testing for this update